index.js

```diff
@@ -551,13 +551,17 @@ const runTestsUsingCommandLine = async () => {
   }
 
   if (parallel || group) {
-    const { branch, buildId } = await getCiBuildId()
-    if (branch) {
-      core.exportVariable('GH_BRANCH', branch)
+    var buildIdVar = null
```

Comment:
```suggestion
    let buildIdVar = null
```


Use `let` here to match the rest of the code.